sphinx/source/docs/user_guide/server.rst

```diff
@@ -1019,6 +1019,16 @@ whatever user account is running the Nginx server process. Alternatively, you
 can copy the resources to a global static directory during your deployment
 process.
 
+In order to communicate cookies and headers across processes Bokeh may include
+this information in a JWT token, which is sent across the Websocket. In certain
+cases this token can grow very large and nginx may drop the request. Therefore
```

Comment:
Any known/reproducible particular cases? Is there anything that we could test against, for e.g. benchmarking purpose.

Reply:
I don't think this is something that can should be tested since it's dependent on what exactly the user asked for. The particular case where we were running into this was when adding an OAuth provider (now included by default in Panel), which sets two cookies including the access token and JSON object containing all the user information the OAuth provider (Azure AD) returned. This ends up in the JWT token and gets sent in the websocket protocol header along with other information like the bokeh session ID. That increased the size of the headers to about 16 KB, which is twice the size of the default nginx proxy buffer.